probe-builder/build-probe-binaries

```diff
@@ -328,8 +328,9 @@ function build_probe {
 
 		# While ls output is already sorted alphabetically by default, we need to sort it
 		# by version instead so it doesn't break when we add support for gcc 10.0
-		DOCKERFILE=$(ls $BUILDER_SOURCE/Dockerfile.${BUILDER_DISTRO}-gcc* | sort -V | awk \
-			-v target=$BUILDER_SOURCE/Dockerfile.${BUILDER_DISTRO}-gcc${BUILDER_GCC_MINOR_VERSION} \
+		DOCKERFILE_PREFIX=$BUILDER_SOURCE/Dockerfile.${BUILDER_DISTRO}-gcc
```

Comment:
a comment here would go a long way....even just a real example of a file we'd find in that dir, and what we expect chose_gcc_version to look like. It helps greatly with debugging/grokking down the line...and even better mentioning why we can't just use the OLD method, because if you don't have the context, you'll have no idea why just sorting and taking the first one doesn't work.

Reply:
Any better? :)

Reply:
<3